batch/sql/insert_gpu_resources.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
```

Comment:
I think this is no longer necessary since we're doing automatic resource price discovery now, right?